config/environments/local/settings/common.yaml

```diff
@@ -1,5 +1,6 @@
 env: LOCAL
 cloud: LOCAL
+encryptedFlow: true
```

Comment:
Just a note, for the encrypted flow, we need to bring up the worker binary separately when testing locally right? Is there already documentation around that?

Reply:
Reverted. This is not ready yet.